suite/tests/CMakeLists.txt

```diff
@@ -3585,8 +3585,9 @@ if (BUILD_CLIENTS)
       set(tool.drcacheoff.raw-gzip_expectbase "offline-simple")
     endif ()
     if (liblz4)
```

Comment:
If we're testing no compression, we don't need the liblz4 guard.

Reply:
Done.